ophyd/controls/positioner.py

```diff
@@ -425,8 +421,6 @@ def __init__(self, setpoint, readback=None,
                                         name=self.name))
 
             self._readback.subscribe(self._pos_changed)
-
-            self._set_position(self._readback.value)
```

Comment:
@danielballan Does `ensure_connection` check to see if it's disconnected and raise an exception, or try to connect?

@dchabot I can see where that would seem a bit odd, but in reality the base class shouldn't know anything about the signals implemented in layers above it. So I don't think it's so strange to have to customize it for all subclasses.